### ISSUE TITLE
 UI error: logo not added in pre-release v0.36.1

### ISSUE THREAD

**Issue by thinkverse** [blockstack/blockstack-browser#1903]
`Error while trying to use the following icon from the Manifest: http://localhost:8888/static/images/app-blockstack.png (Download error or resource isn't a valid image)`

Someone forgot to add the Blockstack logo to the v0.36.1 pre-release so the browser tries to fetch something that doesn't exist anymore, so someone that has it, please add it to the static images folder before a proper release.

**Comment by markmhx** [comment]
Thanks @thinkverse for this report! We're meeting to Browser needs tomorrow so will make sure to discuss this bug and assign for resolution quickly.

**Comment by thinkverse** [comment]
Since your here, the favicon doesn't seem to be working either, seems the server isn't returning it even though it's in the folder structure.

**Comment by hstove** [comment]
Hey @thinkverse , can you explain where this image is being loaded? Is this your app's icon?

**Comment by thinkverse** [comment]
No, not mine, it's being requested because of the Blockstack browser's [manifest.json](https://github.com/blockstack/blockstack-browser/blob/master/app/public/manifest.json), inside icons it's requesting;
```javascript
"icons": [{
  "src": "/static/images/app-blockstack.png",
  "sizes": "128x128",
  "type": "image/png"
}]
```
which isn't added to the pre-release build, causing an error to be thrown, after I downladed the image and added it to the programs static folder the error went away, so the [app/images/app-blockstack.png](https://github.com/blockstack/blockstack-browser/tree/master/app/images/app-blockstack.png) is not being pushed to the final build.

**Comment by moxiegirl** [comment]
@markmhx this is still a problem when testing with the local client.  Working with the hello-world the manifest is:

```
{
  "name": "Hello, Blockstack",
  "start_url": "localhost:5000",
  "description": "A simple demo of Blockstack Auth",
  "icons": [{
    "src": "favicon.ico",
    "sizes": "192x192",
    "type": "image/png"
  }]
}
```

This works fine when opening with the webapp:

![image](https://user-images.githubusercontent.com/1347209/62645198-90ae6e80-b900-11e9-95ac-91837ece8fde.png)

But fails when opening the **same** hello-world with our client.

![image](https://user-images.githubusercontent.com/1347209/62645240-a754c580-b900-11e9-8258-93ce6356ea08.png)

The client isn't picking up the icon despite the fact this is our generated sample.

**Comment by markmhx** [comment]
@hstove Is this bug one we can fix implicitly as part of the rebuild?
